frontend/src/components/Layout/Sidebar.component.tsx

```diff
@@ -1,20 +1,31 @@
 import { Menu } from "antd";
+import useGetCurrentUserInfo from "../../hooks/currentUser";
 
 export default function Sidebar() {
-  return (
-    <Menu
-      theme="light"
-      items={[
-        { key: "1", label: "Trip request" },
-        { key: "2", label: "Past Trips" },
-        { key: "3", label: "Notifications" },
-        { key: "4", label: "Department approval requests" },
-        { key: "5", label: "Department statistics" },
-        { key: "6", label: "Expense review requests" },
-        { key: "7", label: "Awaiting payment" },
-        { key: "8", label: "Review trips" },
-        { key: "9", label: "Statistics" },
-      ]}
-    />
-  );
+  const { data } = useGetCurrentUserInfo();
+  const roleList = data?.roles || [];
+
+  const roles = [
+    { key: "1", label: "Trip request" },
+    { key: "2", label: "Past Trips" },
+  ];
+
+  if (roleList.includes("DEPARTMENT_HEAD")) {
+    roles.push({ key: "4", label: "Department approval requests" });
+    roles.push({ key: "5", label: "Department statistics" });
+  }
+
+  if (roleList.includes("ACCOUNTANT")) {
+    roles.push({ key: "6", label: "Expense review requests" });
```

Comment:
Posto vec imamo sve rute na FE mozemo vec polinkat ovaj sidebar sa pageovima tako da se moze kliktat i mijenjat url

U label moze stavit `<Link to={expenseReviewRequestsRoute.to}> Expense review requests </Link>`

Reply:
7395b18b010e152328fa0966b816cd89a7293622